main.go

```diff
@@ -14,18 +14,20 @@ import (
 )
 
 type userFlags struct {
-	outFile string
-	pkgName string
-	args    []string
+	outFile   string
+	pkgName   string
+	formatter string
+	args      []string
 }
 
 func main() {
 	var flags userFlags
 	flag.StringVar(&flags.outFile, "out", "", "output file (default stdout)")
 	flag.StringVar(&flags.pkgName, "pkg", "", "package name (default will infer)")
+	flag.StringVar(&flags.formatter, "fmt", "", "go pretty-printer: gofmt (default) or goimports")
 
 	flag.Usage = func() {
-		fmt.Println(`moq [flags] destination interface [interface2 [interface3 [...]]]`)
+		fmt.Println(`moq [flags] source-dir interface [interface2 [interface3 [...]]]`)
```

Comment:
See my note about `source-dir` above. Maybe also extend the usage string for better clarity.

Reply:
@matryer What's your opinion on extending the usage string? How it could look:
```
moq [flags] source-dir interface [interface2 [interface3 [...]]]

source-dir is the path to directory where the interface is defined.

Flags:
  -fmt string
    	go pretty-printer: gofmt (default) or goimports
  -out string
    	output file (default stdout)
  -pkg string
    	package name (default will infer)

Specifying an alias for the mock is also supported with the format 'interface:alias'
Ex: moq -pkg different . MyInterface:MyMock
```